kore/test/Test/Kore/Rewrite/Axiom/Matcher.hs

```diff
@@ -1078,6 +1079,13 @@ test_matching_Set =
         (mkVar sSet)
         (mkSet [mkElemVar xInt] [])
         [(sSet, mkSet [mkElemVar xInt] [])]
+    , matches
+        "[x:Int] s:Set matches [1] s':Set"
+        (mkSet [mkElemVar xInt] [mkVar sSet])
+        (mkSet [mkInt 1] [mkVar s'Set])
+        [ (inject xInt, mkInt 1)
+        , (sSet, mkSet [] [mkVar s'Set])
+        ]
```

Comment:
This was actually passing before, I thought the bug was here before I traced the execution